molgenis-data-elasticsearch/src/test/java/org/molgenis/data/elasticsearch/index/IndexBootstrapperTest.java

```diff
@@ -91,7 +93,8 @@ public void testStartupFailedIndexJobs()
 		when(action.getEntityId()).thenReturn("myEntityId");
 
 		when(dataService.findAll(IndexJobExecutionMeta.INDEX_JOB_EXECUTION,
-				new QueryImpl<IndexJobExecution>().eq(JobExecutionMetaData.STATUS, "FAILED"), IndexJobExecution.class))
+				new QueryImpl<IndexJobExecution>().eq(JobExecutionMetaData.STATUS, FAILED).or().eq(JobExecutionMetaData.STATUS, PENDING)
+						.or().eq(JobExecutionMetaData.STATUS, RUNNING).or().eq(JobExecutionMetaData.STATUS, CANCELED), IndexJobExecution.class))
```

Comment:
RUNNING should already be handeled by https://github.com/molgenis/molgenis/blob/d6d355d9196ea28ed5924f18e0c6ba9b014c0db8/molgenis-jobs/src/main/java/org/molgenis/data/jobs/JobBootstrapper.java
right?